src/timepicker/timepicker.js

```diff
@@ -51,14 +51,14 @@ angular.module('ui.bootstrap.timepicker', [])
   var hourStep = timepickerConfig.hourStep;
   if ($attrs.hourStep) {
     $scope.$parent.$watch($parse($attrs.hourStep), function(value) {
-      hourStep = parseInt(value, 10);
+      hourStep = parseInt(+value, 10);
```

Comment:
Oh I meant just changing it all to `hourStep = +value` and all the rest accordingly. No need to run `parseInt` after `+`, it's already a number at that stage.